src/Maestro/Maestro.Data/Migrations/20251125150730_AddNamespace.Designer.cs

```diff
@@ -0,0 +1,1212 @@
+﻿// <auto-generated />
```

Comment:
Not sure this won't need the license header

Reply:
looks like we don't have tit on any other `Designer` file. Also, VS not complaining about it, so I think it's not needed